pkgs/by-name/ca/candy-icons/package.nix

```diff
@@ -30,6 +31,11 @@ stdenvNoCC.mkDerivation {
     runHook postInstall
   '';
 
+  passthru.updateScript = nix-update-script {
```

Comment:
use unstableGitUpdater, it's simpler and out of the box

Reply:
Missed the [update scripts](https://wiki.nixos.org/wiki/Nixpkgs/Update_Scripts) article in nixos wiki. Thanks for the tip!